shapeshift/web
# feat: add a PriceChart component to FoxPage

## Description
We need to use a PriceChart component, like on the asset page, but depending on which asset tab we selected on Fox page.

I used every components we had to do this. I used a selector in order to select the marketData for the `PriceChart`component


## Notice




## Pull Request Type


## Issue (if applicable)
closes #1795


## Risk
- no risk


## Testing
- Go to Fox page
- See if charts are well displayed


## Screenshots (if applicable)
![image](https://user-images.githubusercontent.com/14963751/169660671-ec13b6b9-8688-414f-98ba-6aeae67fe618.png)
